mongodump/mongodump_test.go

```diff
@@ -67,6 +67,7 @@ func simpleMongoDumpInstance() *MongoDump {
 			Connection: connection,
 			Auth:       &auth,
 			Verbosity:  &options.Verbosity{},
+			URI:        &options.URI{},
```

Comment:
I added this because the mongodump tests panicked because opts.URI.ParsedConnString() doesn't check if the URI is nil.

Reply:
Yeah, for things like this is why I rewrote the mongorestore tests to go through the whole command line arg parsing rather than manually setting the options like this. Maybe sometime in the future we'll get around to updating the rest of our integration tests.